src/components/CurrentDirectors/CurrentDirectors.vue

```diff
@@ -0,0 +1,157 @@
+              </v-tooltip>
+            </v-col>
+
+            <!-- Mailing Address -->
+            <v-col class="pr-5">
+              <base-address class="director-detail" :address="orgPerson.mailingAddress" />
+            </v-col>
+
+            <!-- Delivery Address -->
+            <v-col class="pr-5">
+              <p v-if="isSame(orgPerson.mailingAddress, orgPerson.deliveryAddress)"
+                class="director-detail">Same as Mailing Address
+              </p>
+              <base-address v-else class="director-detail" :address="orgPerson.deliveryAddress"/>
+            </v-col>
+
+            <!-- Appoinment Date -->
+            <v-col class="pr-5">
+              <span class="director-detail">{{ orgPerson.roles[0].appointmentDate }} to Current</span>
+            </v-col>
+          </v-row>
+        </div>
+      </div>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Mixins } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { OrgPersonIF } from '@/interfaces'
+import { RoleTypes } from '@/enums'
+import { CommonMixin } from '@/mixins'
+import BaseAddress from 'sbc-common-components/src/components/BaseAddress.vue'
+
+@Component({
+  components: {
+    BaseAddress
+  }
+})
+export default class CurrentDirectors extends Mixins(CommonMixin) {
+  // Declarations for template
+  readonly RoleTypes = RoleTypes
+
+  // Global getters
+  @Getter getPeopleAndRoles!: OrgPersonIF[]
+
+  /** Headers for the person table. */
+  readonly tableHeaders = ['Name', 'Mailing Address', 'Delivery Address', 'Effective Dates']
+
+  /**
+   * Called when component is mounted.
+   */
+  private mounted (): void {}
+
+  /**
+   * Returns the current directors from the people and roles.
+   * @returns array of directors
+   */
+  private get currentDirectors () : OrgPersonIF[] {
+    const directors = this.getPeopleAndRoles
+      .filter(people => people.roles.some(role =>
+        role.roleType.toLowerCase() === 'director' && role.cessationDate === null))
+    for (const director of directors) {
+      director.roles = director.roles.filter(role => role.roleType.toLowerCase() === 'director')
+    }
+    return directors
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+.header-container {
+  display: flex;
+  background-color: $BCgovBlue5O;
+  padding: 1.25rem;
+}
+
+[class^="col"] {
+  padding-top: 0;
+  padding-bottom: 0;
+}
+
+.instructional-text {
+  font-size: 16px;
+  color: $gray7;
+}
+
+.people-roles-list-header {
+  box-shadow: 0px 2px $gray1;
+}
+
+.people-roles-content {
+  border-top: 1px solid $gray1;
+  font-size: 0.875rem;
+}
+
+.directors-title {
+  font-size: 14px;
+  color: $gray9;
+  font-weight: bold;
+}
+
+.director-name {
+  font-size: 16px;
+  text-transform: capitalize;
+  color: $gray9;
+  font-weight: bold;
+}
+
+.director-detail {
+  font-size: 14px;
```

Comment:
Could be converted to .875rem

Reply:
Yes, please specify font sizes in rem.
1 rem = 16px

https://www.google.com/search?q=why+use+rem+instead+of+px

Reply:
Please fix font sizes in CurrentDirectors.vue. See lines 126 and 131.

Reply:
@jachurchill ^^